app/javascripts/main/fileUtils.js

```diff
@@ -1,117 +1,124 @@
-var { app } = require('electron');
-var fs = require('fs');
-var path = require('path');
-var request = require("request");
-var appPath = app.getPath('userData');
+import yauzl from "yauzl";
```

Comment:
Should be single quotes

Reply:
Be sure to single quote all the things :)

Reply:
Gotcha, I'll make a prettier config to make sure that doesn't happen again :)